packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -30,6 +29,7 @@ const LOG_CONTEXT = 'AddJob';
 export class AddJob {
   constructor(
     private jobRepository: JobRepository,
+    @Inject(forwardRef(() => QueueService))
```

Comment:
Forced to implement this. Need to investigate what's the circular dependency causing this.